android/build.gradle

```diff
@@ -28,7 +28,7 @@ buildscript {
 apply plugin: 'com.android.library'
 
 def DEFAULT_PLAY_SERVICES_BASE_VERSION = '16.0.1'
-def READER_SDK_VERSION = '[1.2.1, 2.0)'
+def READER_SDK_VERSION = '[1.3.0, 2.0)'
```

Comment:
should we upgrade to latest?

Reply:
I think we should since we've moved to AndroidX which is supported in 1.3.1 only.

Reply:
Copy that, bumping up to `1.3.1`